examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+
+    show_anatomical_slices(seed_roi_img.get_fdata(), 'Seed ROI')
+
+    tractogram = aft.track(dti_params['params'], seed_mask=seed_roi,
+                           stop_mask=FA_data, stop_threshold=0.1)
+    save_tractogram(tractogram, op.join(working_dir, 'dti_streamlines.trk'),
+                    bbox_valid_check=False)
+
+    create_density_map(tractogram, 'afq_dti_density_map.nii.gz')
+else:
+    tractogram = load_tractogram(op.join(working_dir, 'dti_streamlines.trk'),
+                                 img)
+
+tractogram.to_vox()
+
+##########################################################################
+# Segmentation:
+# -------------
+# In this stage, streamlines are tested for several criteria: whether the
+# probability that they belong to a bundle is larger than a threshold (set to
+# 0, per default), whether they pass through inclusion ROIs and whether they do
+# not pass through exclusion ROIs.
+#
+# .. note::
+#   By default segmentation:
+#
+#   - uses Streamlinear Registration algorithm
+#
+#   - does not clip streamlines to be between ROIs
+#
+#   - All b-values less than or equal to 50 are considered to be
+#     without diffusion weighting.
+#
+# 
+# Segmentation will result in a `fiber_group` for each bundle, which
+# contains the following keys:
+#
+#    - `sl`
+#
+#       StatefulTractogram encompassing the selected streamlines
+#
+#    - `idx`
+#
+#       indexes to selected streamlines
+#
+# .. note::
+#    Currently it is not possible to define endpoint filters for novel bundles,
+#    but this is something we expect to address. However we can run
+#    segmentation by ignoring endpoint filters. This means that additional
+#    streamlines may be included that would otherwise be excluded.
+
+
+tractogram_img = nib.load(op.join(working_dir, 'afq_dti_density_map.nii.gz'))
+show_anatomical_slices(tractogram_img.get_fdata(), 'DTI Density Map')
+
+print("Segmenting fiber groups...")
+
+segmentation = seg.Segmentation(return_idx=True,
+                                filter_by_endpoints=False)
+
+segmentation.segment(bundles,
+                     tractogram,
+                     fdata=hardi_fdata,
+                     fbval=hardi_fbval,
+                     fbvec=hardi_fbvec,
+                     mapping=mapping,
+                     reg_template=MNI_T2_img)
+
+fiber_groups = segmentation.fiber_groups
+
+for bundle in bundles:
+    tractogram = StatefulTractogram(fiber_groups[bundle]['sl'].streamlines,
+                                    img,
+                                    Space.VOX)
+    tractogram.to_rasmm()
+    save_tractogram(tractogram, op.join(working_dir, f'afq_{bundle}_seg.trk'),
+                    bbox_valid_check=False)
+
+    create_density_map(tractogram, f'afq_{bundle}_seg_density_map.nii.gz')
+    tractogram_img = nib.load(op.join(working_dir,
+                                      f'afq_{bundle}_seg_density_map.nii.gz'))
+    show_anatomical_slices(tractogram_img.get_fdata(),
```

Comment:
Any final thoughts for the various streamline density maps (tracking, segmentation, and cleaned)